glide-core/tests/test_client.rs

```diff
@@ -328,11 +367,65 @@ pub(crate) mod shared_client_tests {
             .await;
 
             let mut cmd = redis::Cmd::new();
-            cmd.arg("BLPOP").arg("foo").arg(0); // 0 timeout blocks indefinitely
+            cmd.arg("BLPOP").arg("foo").arg(0.3); // server should return null after 300 millisecond
+            let result = test_basics.client.send_command(&cmd, None).await;
+            assert!(result.is_ok());
+            assert_eq!(result.unwrap(), Value::Nil);
+        });
+    }
+
+    #[rstest]
+    #[timeout(SHORT_CLUSTER_TEST_TIMEOUT)]
+    fn test_blocking_command_with_zero_timeout_blocks_indefinitely(
+        #[values(false, true)] use_cluster: bool,
+    ) {
+        // We test that when blocking command is passed with 0 as the timeout duration, the command will be blocked indefinitely
```

Comment:
mismatch on comments
```suggestion
        // We test that when blocking command is passed with a negative timeout the command will return with an error
```